app/controllers/api/verify/complete_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Api::Verify::CompleteController < Api::BaseController
+  before_action :confirm_two_factor_authenticated_for_api
+
+  def create
+    result = Api::ProfileCreationForm.new(
+      user_password: verify_params[:password],
+      user_bundle: verify_params[:details],
+      user_session: user_session,
+    ).submit
+    analytics.track_event(Analytics::IDV_PERSONAL_KEY_VISITED, result.to_h)
```

Comment:
Hm, I feel like these sorts of view events ("page visited") should be handled by the UI pieces, rather than the verification API. We'll probably run into challenges with feature parity around our [front-end logging prefix](https://github.com/18F/identity-idp/blob/db0eb461b5a979f9d0c2f535153cec49c12d705c/app/controllers/frontend_log_controller.rb#L9), but maybe we can allowlist certain events like these to log from the React app.

Which is all to say: We can probably remove it here, and do a pass at events separately.

```suggestion
```

Reply:
Also, based on what I'm seeing in `ProfileCompletionForm#extra_attributes`, we definitely wouldn't want to be logging the result, since it contains the personal key, which is essentially a sort of password.

Reply:
Probably a separate convo, but there's a pattern in the codebase of returning sensitive values inside form responses... and hoping that we remember to remove them before logging the whole form response... I would rather we just have separate return values for sensitive values like that if we are able to, or maybe like a `form_response, sensitive_value = form.submit` or something